test/integration/api-support/next.config.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  target: 'serverless'
+  target: 'server'
```

Comment:
Why was this switched? Can we revert it?

Reply:
I want to do it properly with [serverless build](https://github.com/zeit/next.js/pull/7649) when this lands